data/sql/updates/pending_db_world/rev_1612257567543292300.sql

```diff
@@ -0,0 +1,17 @@
+DELETE FROM `spell_dbc` WHERE (`ID`=57946);
+INSERT INTO `spell_dbc` VALUES ('57946', '0', '0', '0', '65536', '1024', '33554432', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '1', '0', '0', '8', '0', '0', '0', '101', '0', '16', '6', '6', '0', '-2', '0', '0', '0', '0', '1', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '-1', '0', '0', '3', '0', '0', '1', '0', '0', '0', '0', '0', '26', '0', '0', '0', '0', '0', '1', '0', '0', '0', '0', '0', '28', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '1225', '0', '208', '0', '0', 'Life Tap', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '16712190', 'Rank 1', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '16712190', 'Converts ${$m1} health into ${$m1*$<mult>+$SPS*.5*$<mult>} mana.  Spell power increases the amount of mana returned.\r\n', 'Converts ${$m1} health into ${$m1*$<mult>+$SPS*.5*$<mult>} mana.  Spell power increases the amount of mana returned.', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '16712190', '', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '0', '16712190', '0', '133', '1500', '0', '5', '262144', '0', '0', '0', '1', '1', '0', '1065350000', '1065350000', '1065350000', '0', '0', '0', '0', '0', '0', '32', '0', '0', '0', '0', '0', '0', '175', '0');
```

Comment:
you don't need multiple DELETE/INSERT statements, you can just use one as explained in: https://www.azerothcore.org/wiki/Dealing-with-SQL-files

Reply:
Also dont use ' around numbers